client/src/components/MagicalFactor/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Keywords from '../../assets/Keywords';
+
+import './style.css';
+
+const MagicalFactor = ({ value: { text, src }, reviewText }) => (
+  <div className="magical">
+    <div className="magical__keyword">
+      <div className="magical__keyword__img">
+        <img src={Keywords[src]} alt={text} className="magical__keyword__img" />
+      </div>
+      <div
+        className={`magical__keyword__div${text.length > 18 ? ' smaller' : ''}`}
+      >
+        <p>{text}</p>
+      </div>
+    </div>
+    <div className="magical__review">
+      <p className="magical__review__p">{reviewText}</p>
+    </div>
+  </div>
+);
+
+MagicalFactor.propTypes = {
+  value: propTypes.shape({
+    text: propTypes.string.isRequired,
+    src: propTypes.string.isRequired,
+  }).isRequired,
+  reviewText: propTypes.string.isRequired,
```

Comment:
in the typeform, it is required
![Screenshot from 2019-10-29 09-45-56](https://user-images.githubusercontent.com/8256872/67747521-90220180-fa31-11e9-86d0-6338f966d494.png)

Reply:
that does not mean it is correct or the right/best way to do it. From user's perspective, I don't think it is a good idea to force users to write explanation for every characteristic - I know it would turn me off from using it. Generally I personally would prefer to have just one text field for written review (like for example booking.com has).

If your client requested it this way, don't be afraid to push back and try to persuade him otherwise - ideally you would support your position with some data - best would be an A-B testing on real users, which of course is not feasible with your scope, but you can mention that if filling the form will take too long it will result in user dropoff (and maybe showing some study that proves it)

Reply:
Yes i do get your point, but she have thought about that, and we discussed it briefly, and she wants it to be this way, thats why she used typeform in the first place, so it could be a bit intuitive and more non-traditional way